deploy/README.md

```diff
@@ -1,31 +1,180 @@
+Considering that password is **@rrebol**. The content of the postgres.env file would be:
 
-  ```
-  sudo bash deploy-stack.sh
-  ```
+```
+POSTGRES_PASSWORD=@arrebol
+```
+
+#### PgAdmin Configuration
+
+File: pgadmin.env
+
+```
+PGADMIN_DEFAULT_EMAIL=
+PGADMIN_DEFAULT_PASSWORD=
+```
+
+The **PGADMIN_DEFAULT_EMAIL** and **PGADMIN_DEFAULT_PASSWORD** define a admin user to login in the pgadmin. 
+The **PGADMIN_DEFAULT_EMAIL** does not necessarily have to exist but it must be valid.
+
+Considering that **PGADMIN_DEFAULT_EMAIL** is **arrebol@lsd.ufcg.edu.br** and **PGADMIN_DEFAULT_PASSWORD** is **pg@dmin**. The content of the pgadmin.env file would be:
+
+```
+PGADMIN_DEFAULT_EMAIL=arrebol@lsd.ufcg.edu.br
+PGADMIN_DEFAULT_PASSWORD=pg@dmin
+```
+
+#### Application properties Configuration
+
+File: application.properties
+
+There is one property that need to be filled in this file:
+* **spring.datasource.password**
+
+The **spring.datasource.password** is the password that arrebol service was use to connect to data base. The value of it is the **POSTGRES_PASSWORD** that was define previously.
+
+Considering that **POSTGRES_PASSWORD** is **@rrebol**. The content of the application.properties file would be:
+
+```
+spring.jpa.database=POSTGRESQL
+spring.datasource.driverClassName=org.postgresql.Driver
+spring.datasource.platform=postgres
+spring.datasource.url=jdbc:postgresql://postgresql:5432/arrebol
+spring.datasource.username=postgres
+spring.datasource.password=@rrebol
+spring.jpa.generate-ddl=true
+
+spring.jpa.hibernate.ddl-auto=update
+```
+
+#### Arrebol Properties
+
+File: arrebol.json
+```json
+{
+  "poolType":"docker",
+  "properties": [
+    {
+      "key":"workerPoolSize",
+      "value": 5
+    },
+    {
+      "key": "imageId",
+      "value": "wesleymonte/simple-worker"
+    },
+    {
+      "key": "resourceAddresses",
+      "value": []
+    }
+  ]
+}
+```
+
+| Field                             | Description    |
+|:---------------------------------:|----------------|
+| **workerPoolSize**                |  Defines the maximum number of workers that can run on the same node. |
+| **imageId**                       | Sets the default docker image used to create workers. |
+| **resourceAddress**               | Defines an address list of worker nodes. |
+
+The workerPoolSize is **5** and imageId is **wesleymonte/simple-worker** by default. 
+
+Considering that your worker nodes are **10.30.1.1** and **10.30.1.2**. The content of the **arrebol.json** file would be:
+
+```json
+{
+  "poolType":"docker",
+  "properties": [
+    {
+      "key":"workerPoolSize",
+      "value": 5
+    },
+    {
+      "key": "imageId",
+      "value": "wesleymonte/simple-worker"
+    },
+    {
+      "key": "resourceAddresses",
+      "value": ["10.30.1.1", "10.30.1.2"]
+    }
+  ]
+}
+```
+
+### 4.Install
+
+Log in the arrebol host and run the below commands to install the arrebol service:
```

Comment:
Maybe still in the arrebol host is better. I think the user hasn't logout yet...

Reply:
Done.